torch_harmonics/convolution.py

```diff
@@ -257,7 +172,7 @@ def _precompute_convolution_tensor_s2(
         phi = torch.arctan2(y, x) + torch.pi
 
         # find the indices where the rotated position falls into the support of the kernel
-        iidx, vals = kernel_handle(theta, phi)
+        iidx, vals = filter_basis.compute_support_vals(theta, phi, r_cutoff=theta_cutoff)
```

Comment:
The naming is weird I think, since radial and angular cutoff are technically different. Can we streamline that somehow?